src/lib/css-tag.ts

```diff
@@ -12,8 +12,8 @@ found at http://polymer.github.io/PATENTS.txt
 /**
  * Whether the current browser supports `adoptedStyleSheets`.
  */
-export const supportsAdoptingStyleSheets =
-    ('adoptedStyleSheets' in Document.prototype) &&
+export const supportsAdoptingStyleSheets = (window.ShadowRoot) &&
+    ('adoptedStyleSheets' in ShadowRoot.prototype) &&
```

Comment:
We need to keep this checks against Document, since we have developers that delete the properties on Document.prototype to disable the use of adopted style sheets so they can edit in DevTools.

Reply:
Sure. I've reverted this but kept the check for `ShadowRoot`.